core/src/main/java/de/danoeh/antennapod/core/preferences/UserPreferences.java

```diff
@@ -141,6 +141,7 @@ private UserPreferences(){}
     public static final int EPISODE_CLEANUP_NULL = -2;
     public static final int EPISODE_CLEANUP_EXCEPT_FAVORITE = -3;
     public static final int EPISODE_CLEANUP_DEFAULT = 0;
+    public static final String PREF_REPEAT_EPISODE = "RepeatEpisode";
```

Comment:
Please stay consistent with most other settings

```suggestion
    public static final String PREF_REPEAT_EPISODE = "prefRepeatEpisode";
```